outputs.tf

```
@@ -0,0 +1,4 @@
+output "zones" {
+  description = "Created zone identificators"
```

```suggestion
  description = "Created zone identificators"
```

Is "identificators" the appropriate word here?